modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+=== Compatibility and support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. +
```

Comment:
Not sure if we need the extra + at the end of the sentence. Strangely, I do not see it in the rendered version